src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,170 @@
 package core.basesyntax;
 
+//import jdk.internal.util.ArraysSupport;
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
+
 public class ArrayList<T> implements List<T> {
+
+
+    private static final int def_capacity = 10;
```

Comment:
Should be renamed using CONSTANT_CASE. See [here](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names).

Also, don't use `def`, instead use the full word `default`.
```suggestion
    private static final int DEFAULT_CAPACITY = 10;
```